### PR TITLE
Update g.gisenv.html

### PR DESCRIPTION
**Pull request by jidanni**
https://trac.osgeo.org/grass/ticket/3997

### CODE REVIEW

File: general/g.gisenv/g.gisenv.html [hunk 1]
```diff
@@ -145,7 +145,7 @@ <h2>NOTES</h2>
 By default the GRASS variables are stored in <em>gisrc</em> file
 (defined by environmental
 variable <em>GISRC</em>). If <b>store=mapset</b> is given than the
-variables are stored in <tt>$GISDBASE/$LOCATION_NAME/$MAPSET/VAR</tt>.
+variables are stored in <tt>$GISDBASE/$LOCATION_NAME/$MAPSET/VAR</tt> after the current GRASS session is closed.
```
Comment (neteler) [inline general/g.gisenv/g.gisenv.html]:
```suggestion
variables are stored in <tt>$GISDBASE/$LOCATION_NAME/$MAPSET/VAR</tt>
after the current GRASS session is closed.
```
(breaking long lines as per our HTML style guide)

Comment (jidanni) [reply general/g.gisenv/g.gisenv.html]: Maybe also mention "and stored in a temporary file during the current session."

**Comment by metzm** [comment]
This is an error in the `g.gisenv` manual. It does **not** store variables in `$GISDBASE/$LOCATION_NAME/$MAPSET/VAR`. This file only stores information about the default vector database connection, which can be modified for the current mapset with `db.connect.`

File: general/g.gisenv/g.gisenv.html [hunk 2]
```diff
@@ -145,7 +145,8 @@ <h2>NOTES</h2>
 By default the GRASS variables are stored in <em>gisrc</em> file
 (defined by environmental
 variable <em>GISRC</em>). If <b>store=mapset</b> is given than the
```
Comment (neteler) [inline general/g.gisenv/g.gisenv.html:147]:
```suggestion
variable <em>GISRC</em>). If <b>store=mapset</b> is given then the
```

File: general/g.gisenv/g.gisenv.html [hunk 3]
```diff
@@ -145,7 +145,8 @@ <h2>NOTES</h2>
...
-variables are stored in <tt>$GISDBASE/$LOCATION_NAME/$MAPSET/VAR</tt>.
+variables are stored in <tt>$GISDBASE/$LOCATION_NAME/$MAPSET/VAR</tt>
```
Comment (neteler) [inline general/g.gisenv/g.gisenv.html:148]:
```suggestion
variables are stored in <tt>&lt;gisdbase&gt;/&lt;location&gt;/&lt;mapset&gt;/VAR</tt>
```

**Comment by neteler** [comment]
> This is an error in the `g.gisenv` manual. It does **not** store variables in `$GISDBASE/$LOCATION_NAME/$MAPSET/VAR`. This file only stores information about the default vector database connection, which can be modified for the current mapset with `db.connect.`

Should the current proposed modification be removed then?

**Comment by metzm** [comment]
> > This is an error in the `g.gisenv` manual. It does **not** store variables in `$GISDBASE/$LOCATION_NAME/$MAPSET/VAR`. This file only stores information about the default vector database connection, which can be modified for the current mapset with `db.connect.`
> 
> Should the currently proposed modification be removed then?

No, I was mistaken, `g.gisenv` can indeed use `$GISDBASE/$LOCATION_NAME/$MAPSET/VAR` to read/write variables. The change is fine with me.

**Review by metzm — APPROVED** [review]

**Comment by neteler** [comment]
(Close/reopen to wake up flake8 test)

**Pull request merged**